x-pack/libbeat/management/manager_test.go

```diff
@@ -72,3 +73,24 @@ type dummyReloadable struct{}
 func (dummyReloadable) Reload(config *reload.ConfigWithMeta) error {
 	return nil
 }
+
+type dummyReloadableList struct{}
+
+func (dummyReloadableList) ReloadableList() error {
+	return nil
+}
+
+func dummyCM() *Manager {
+	// Use the 3 register point of Filebeat.
+	reg := reload.NewRegistry()
+	reg.Register("output", &dummyReloadable{})
+	reg.Register("filebeat.inputs", &dummyReloadable{})
+	reg.Register("filebeat.modules", &dummyReloadable{})
+
+	cm := &Manager{
+		registry: reg,
+		logger:   logp.NewLogger("test"),
+	}
+
+	return cm
+}
```

Comment:
[Question]
Whereas I understand the utility of having a dummy Manager for testing, I don't see any usage for them, so I wonder if it's necessary to add it now.

Reply:
I'm a fan of adding mocks when possible. Since this will potentially be consumed by someone writing a new beat, they might just grep for a mock interface to throw into their tests.